packages/core/admin/server/bootstrap.js

```diff
@@ -20,6 +20,17 @@ const registerAdminConditions = () => {
   getService('permission').conditionProvider.registerMany(adminConditions.conditions);
 };
 
+const registerModelHooks = () => {
+  const { sendDidChangeInterfaceLanguage } = getService('metrics');
+
+  strapi.db.lifecycles.subscribe({
+    models: ['admin::user'],
+    afterCreate: sendDidChangeInterfaceLanguage,
+    afterUpdate: sendDidChangeInterfaceLanguage,
+    afterDelete: sendDidChangeInterfaceLanguage,
```

Comment:
I'm not sure the wording of the event/function really explains what's going on, here you'll send the `sendDidChangeInterfaceLanguage` each time an admin user has been created/updated/deleted even if its interface language didn't change.
Although it could be understandable for creation/deletion as we always need to trigger this event, it becomes harder to justify when looking at an email update (which here, will trigger the event anyway).

Is it something that has been discussed internally?

A simple idea to minimize this side effect: Since we don't support (at least for now) the field-based hooks, you could return early when the `afterUpdate`'s input payload doesn't contain a "preferedLanguage" value, what do you think?